essentials-framework/Essentials DM/Essentials_DM/Endpoints/Transmitters/DmTx201CController.cs

```diff
@@ -195,6 +196,28 @@ public override bool CustomActivate()
 			return base.CustomActivate();
 		}
 
+        /// <summary>
+        /// Enables or disables free run
+        /// </summary>
+        /// <param name="enable"></param>
+        public void SetFreeRunEnabled(bool enable)
+        {
+            if (enable)
```

Comment:
//style/conciseness: this could be written as
Tx.VgaInput.FreeRun = enable ? eDmFreeRunSetting.Enabled : eDmFreeRunSetting.Disabled;